modules/swagger-codegen/src/main/resources/typescript-angular/api.service.mustache

```diff
@@ -175,6 +175,11 @@ export class {{classname}} {
             '{{{mediaType}}}'{{#hasMore}},{{/hasMore}}
             {{/consumes}}
         ];
+
+        if ((consumes != null) && (consumes.length > 0)) {
+            headers.set('Content-Type', consumes.join(';'));
```

Comment:
Just a little code style change: could you please remove the braces around `consumes != null` and `consumes.length > 0`? (same below in line 202). Thanks for the PR.